content/blog/2018-10-02-outcomerate.md

```diff
@@ -0,0 +1,60 @@
+---
+slug: "outcomerate"
+title: Honest Assessments of Survey Quality with AAPOR Outcome Rates
```

Comment:
Is there a way you could modify the title to reveal more of what `outcomerate` does? i.e. I didn't know what AAPOR was but seeing the "public opinion research" part of American Association of Public Opinion Research intrigued me. Without that, I didn't have a sense that this was social sciences related (we have a growing community of people working in R for social sciences) or what type of outcome rates it deals with (ecological field research maybe ;-) ?)

I also note that the definition of AAPOR does not come till second section "What Problem Does the Package Solve?" which makes it more impt to try to get the info out earlier, and in title if possible

Reply:
Point very well taken. What do you think about _"outcomerate: Transparent Communication of Quality in Social Surveys"_ instead?